src/test/java/io/github/mfvanek/pg/model/index/DuplicatedIndexesTest.java

```diff
@@ -198,14 +203,17 @@ void newFactoryConstructor() {
         assertThatThrownBy(() -> DuplicatedIndexes.of(null, null))
                 .isInstanceOf(NullPointerException.class)
                 .hasMessage("tableName cannot be null");
-        assertThatThrownBy(() -> DuplicatedIndexes.of(IndexWithSize.of("t", "i1", 1L), null))
+
+        final IndexWithSize indexWithSize = IndexWithSize.of("t", "i1", 1L);
+        assertThatThrownBy(() -> DuplicatedIndexes.of(indexWithSize, null))
                 .isInstanceOf(NullPointerException.class)
                 .hasMessage("secondIndex cannot be null");
+
+        final IndexWithSize firstIndex = IndexWithSize.of("t", "i1", 1L);
+        final IndexWithSize secondIndex = IndexWithSize.of("t", "i2", 2L);
+        final IndexWithSize[] otherIndexes = {null, IndexWithSize.of("t", "i4", 4L)};
```

Comment:
Why not `final IndexWithSize fourthIndex = IndexWithSize.of("t", "i4", 4L)` ?

Reply:
Array was deleted and two separate variables were created